kolibri/plugins/coach/assets/src/routes/groupsRoutes.js

```diff
@@ -1,24 +1,49 @@
 import store from 'kolibri.coreVue.vuex.store';
 import { PageNames } from '../constants';
-import ReportsGroupListPage from '../views/reports/ReportsGroupListPage.vue';
+import { useGroups } from '../composables/useGroups';
+import GroupsPage from '../views/plan/GroupsPage';
+import GroupEnrollPage from '../views/plan/GroupEnrollPage';
+import GroupMembersPage from '../views/plan/GroupMembersPage';
 
 const CLASS = '/:classId';
 const ALL_GROUPS = '/groups';
+const GROUP = '/groups/:groupId';
 
 function path(...args) {
   return args.join('');
 }
 
+const { showGroupsPage } = useGroups();
+
 export default [
   {
     name: PageNames.GROUPS_ROOT,
     path: path(CLASS, ALL_GROUPS),
-    component: ReportsGroupListPage,
-    handler: () => {
-      store.dispatch('notLoading');
```

Comment:
Oh its because we already handle this in the `showGroupsPage` https://github.com/learningequality/kolibri/blob/87418dd7b5dedf0a09213b8e209d61cc341b20ef/kolibri/plugins/coach/assets/src/composables/useGroups.js#L22 that was needed to show this report groups page.

Reply:
thank you!